data_steward/cdr_cleaner/cleaning_rules/store_pid_rid_mappings.py

```diff
@@ -171,6 +188,14 @@ def validate_rule(self):
     import cdr_cleaner.clean_cdr_engine as clean_engine
 
     ext_parser = parser.get_argument_parser()
+    ext_parser.add_argument(
+        '-e',
+        '--export_date',
+        action='store',
+        dest='export_date',
+        help=('Date of the RDR export. Should adhere to '
```

Comment:
Should we add a type validation on the format of `export_date`?

Reply:
@ChaoPang I considered this and could be swayed to add one.  I initially decided to put the date parsing/validation code into the class initialization so it was checked at one location and dealt with accordingly whether we run the script as a stand alone script or as an import to another module. If the input can't be parsed as a date, the class initializer does raise an error.  I'll see about adding some earlier type checking to it.